frontend/src/constants/Constants.js

```diff
@@ -214,12 +277,17 @@ export const PLACEHOLDER_NOT_APPLICABLE = "N/A";
 export const PLACEHOLDER_SELECT_COMPONENT = "Select component...";
 
 // Units in labels
-export const SECONDS = "(s)";
-export const KILOMETRE = "(km)";
-export const METRE_PER_SECOND = "(m/s)";
-export const YEARS = "(years)";
-export const PERCENTAGE = "(%)";
-export const G_FORCE = "(g)";
+export const SECONDS_UNIT = "(s)";
+export const KILOMETRE_UNIT = "(km)";
+export const METRE_PER_SECOND_UNIT = "(m/s)";
+export const YEARS_UNIT = "(years)";
+export const PERCENTAGE_UNIT = "(%)";
+export const G_FORCE_UNIT = "(g)";
```

Comment:
I'd just call this gravity unit. Force and acceleration are different but this could be confusing.